Vurfan/Inheritance.js

```diff
@@ -0,0 +1,27 @@
+class Employee {
+  constructor(name, age, gender, designation, location, skills) {
+    this.emp_name = name;
+    this.emp_age = age;
+    this.emp_gender = gender;
+    this.epm_designation = designation;
+    this.emp_location = location;
+    this.emp_skills = skills;
+  }
+}
+
+class Comapany extends Employee {
```

Comment:
Is-A test is failing here.

Company is not an employee. Company can have employees though

Reply:
Have modified the code and written in correct pattern .